### DISCUSSION TITLE
Contiuous histogram of column

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1821] [Q&A]
I'm not sure whether the name "Continuous histogram" is the real name of the think I'm looking for. I have a column with a thousand numbers between 0 and 1. I want to look at some kind of plot that'll show me where the highest concentrations of these numbers are, regardless of the order in which they appear.

I guess this is a histogram, except maybe I want the resolution of the bars to be customizable.

Any idea how I could do such a visualization with VisiData?

**Comment by saulpw** [comment]
Try turning on the `numeric_binning` option and type the column as a float.  It should bucketize by some sort of reasonable range.

**Reply by cool-RR** [reply]
I hid all columns except the column I want. I pressed `z Shift-o` and set `numeric_binning` to `True`. I quit the options sheet but I didn't see any consequence of that. I pressed `.` to draw a plot of that column but that plot was completely normal, showing the numbers by order.

**Reply by cool-RR** [reply]
Nudge.

**Reply by saulpw** [reply]
I guess I don't understand what you're trying to do here.  I'm planning on starting to host office hours in a couple weeks, this would be a good one to bring by and we can talk through it.

**Reply by cool-RR** [reply]
Note to anyone who found this thread: What I was missing is that after setting `numeric_binning` to `True`, I should do a frequency table `Shift-F` on the column. Then it works well.
